kyaml/yaml/types_test.go

```diff
@@ -166,3 +167,82 @@ type: string
 	}
 	assert.Equal(t, expected, actual)
 }
+
+func TestIsMissingOrNull(t *testing.T) {
+	if IsMissingOrNull(nil) != true {
```

Comment:
Please write this (and all following branch tests) using the usual boolean function check idiom: 
```
if !IsMissingOrNull(nil) {
```
instead of `!= true` or whatever.

Actually, the body of the `IsMissingOrNull` is making a similar idiom mistake (not this PR, but feel free to fix it)